app/build.gradle

```diff
@@ -73,6 +64,8 @@ android {
         buildConfigField 'Boolean', 'USE_AWS_S3', "false"
         buildConfigField 'Boolean', 'ORG_LINK', "false"
 
+        resValue "string", "maps_api_key", "\"${maps_api_key}\""
```

Comment:
Recommended way to set the API key in the manifest file via resources. This also allows to vary the api key between test or prod environment.